### DISCUSSION TITLE
Support GPT Plugins

### DISCUSSION THREAD

**Discussion by lancedouglas1** [allanbunch/node-red-openai-api#21] [Ideas]
First of all, this is a fantastic node-red package.  Thanks so much for making it.

I'd love to be able to trigger gpt plugins via this package.

The example that I'm working on is triggering invideo's video generation plugin like demonstrated in this post: https://community.openai.com/t/unlocking-video-creation-potential-harnessing-invideos-ai-technology/619408

**Comment by allanbunch** [comment]
Great to hear that you're finding the node helpful. We have one of the best ecosystems for developing advanced AI solutions in Node-RED and I'm happy to know that people like you recognize the potential.

Plugin support is a feature that I'd love to integrate though at the moment, Plugins are tightly coupled with ChatGPT proper and OpenAI doesn't yet expose plugin API endpoints.

An alternative approach that might work for you using Node-RED is to use an HTTP Request node in you flow to call [Invideo's plugin API endpoint](https://gptstore.ai/gpts/asX9CMsYPZ-video-ai-by-invideo/actions). I ran a quick test and everything worked as designed. Let me know if you'd like additional context for this approach.
